docker-compose.yml

```diff
@@ -9,6 +9,10 @@ services:
       - default
     volumes:
       - ./data/db:/data/db
+    logging:
+      driver: "syslog"
+      options:
+        syslog-address: "tcp://mozdef_logging_agent:514"
```

Comment:
This seems pretty straight forward, I hope it works as expected.  We can test once the agent image exists.